ctl/waypoint/waypoint.go

```diff
@@ -630,6 +630,8 @@ func getKmeshWaypointImage() string {
 	}
 
 	ver := version.Get().GitVersion
+	// if ver prefix with v, remove it
+	ver = strings.TrimPrefix(ver, "v")
 
 	return fmt.Sprintf("ghcr.io/kmesh-net/waypoint:v%s", ver)
```

Comment:
@silenceper thanks for your contribution!

But why not remove the `v` here?

Reply:
I am not sure whether GitVersion will eventually have the v prefix. If it does not, it will be compatible.